core/src/scrim/scrim.consumer.ts

```diff
@@ -35,8 +35,12 @@ export class ScrimConsumer {
         this.logger.log(`scrim unsuccessful scrimId=${scrimId}`);
         this.logger.log(`scrimId=${scrimId} players didn't check in: ${playersNotCheckedIn.map(p => p.name)}`);
 
-        const initialBanDuration = await this.organizationConfigurationService.getOrganizationConfigurationValue(scrim.organizationId, "queueBanInitialDuration");
-        const durationModifier = await this.organizationConfigurationService.getOrganizationConfigurationValue(scrim.organizationId, "queueBanDurationModifier");
+        const initialBanDuration = await this.organizationConfigurationService.getOrganizationConfigurationValue(scrim.organizationId, OrganizationConfigurationKeyCode.SCRIM_QUEUE_BAN_INITIAL_DURATION_SECONDS);
```

Comment:
`OrganizationConfigurationKeyVaule as ockv` or similar at point of use can make this less verbose.

Reply:
Point of import*